src/tv2-common/migrations/index.ts

```diff
@@ -125,6 +129,49 @@ export function AddGraphicToGFXTable(
 	}
 }
 
+export function mapGFXTemplateToDesignTemplateAndDeleteOriginals(
+	versionStr: string,
+	studio: string,
+	from: string,
+	to: string
+) {
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.mapGFXTemplateToDesignTemplateAndDeleteOriginals.${from}.${studio}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(from) as unknown) as TableConfigItemGFXTemplates[] | undefined
+
+			const designTemplates = (context.getBaseConfig(to) as unknown) as TableConfigItemGfxDesignTemplate[] | undefined
+
+			if (!gfxTemplates || !gfxTemplates.length) {
+				return false
+			}
+
+			if (designTemplates && designTemplates.length) {
+				return false
+			}
+
+			return gfxTemplates.some(template => template.IsDesign)
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(from) as unknown) as TableConfigItemGFXTemplates[]
+			const designTemplates = ((context.getBaseConfig(to) as unknown) as TableConfigItemGfxDesignTemplate[]) ?? []
+
+			gfxTemplates
+				.filter(template => template.IsDesign)
+				.map(template => {
+					designTemplates.push({ ...template, INewsStyleColumn: '' })
+				})
+
+			const newGFXTemplates = gfxTemplates.filter(template => !template.IsDesign)
```

Comment:
Minor thing, but it would be nice if the name were `newGfxTemplates` and not `newGFXTemplates`.